challengeOneWeek3.js

```diff
@@ -1,15 +1,29 @@
 
-    var i;
-    var str='1ab2c3defUIoAu1806';
-    var even =[];
-    var odd =[];
+var str='1ab2c3defUIoAu18x0p6o9i20k100';
+// not yet sure if these should be global
     var numbers = str.match(/\d+/g)
     const chars = str.toLowerCase().match(/[A-z]/g).sort()
```

Comment:
Since you are first converting the string to lowercase, There is no need to match uppercase letters. Using `/a-z/g` will be enough. Another option can be using ignoring the `toLowerCase` method call but using `/a-z/gi`, The i flag makes the regex case insensitive